### ISSUE TITLE
Trying anbox in an environment like aws, but failed with 'Failed to initialize SDL: No available video device'

### ISSUE THREAD

**Issue by firmianavan** [anbox/anbox#1997]
1. Please check that no similar bug is already reported. Have a look on the list of open bugs at https://github.com/anbox/anbox/issues
2. Make sure you are running the latest version of Anbox before reporting an issue. Update snap to latest: `snap refresh --devmode --edge anbox`
3. Make sure you have debug logs enabled:
`sudo snap set anbox debug.enable=true`
4. Reproduce the error while debug logs enabled.
5. Run the anbox logs collection utility and attach the tar file.
`sudo /snap/bin/anbox.collect-bug-info `
[anbox-system-diagnostics-2022-02-11.zip](https://github.com/anbox/anbox/files/8046781/anbox-system-diagnostics-2022-02-11.zip)

6. ** Please paste the result of `anbox system-info` below:**
<details>
<summary>anbox system-info output</summary>

```
$ anbox system-info
version: local-ad377ff
snap-revision: 213
cpu:
  arch:  x86
  brand: AMD EPYC 7K62 48-Core Processor
  features:
    - aes
    - sse4_1
    - sse4_2
    - avx
    - avx2
os:
  name: Ubuntu
  version: 20.04 LTS (Focal Fossa)
  snap-based: true
kernel:
  version: Linux version 5.4.0-99-generic (buildd@lgw01-amd64-007) (gcc version 9.3.0 (Ubuntu 9.3.0-17ubuntu1~20.04)) #112-Ubuntu SMP Thu Feb 3 13:50:55 UTC 2022
  binder: false
  ashmem: true
graphics:
  egl:
    vendor: n/a
    version: n/a
    extensions:
      - n/a
  gles2:
    vendor: n/a
    vendor: n/a
    extensions: []
```
</details>

**Please describe your problem:**
```
$ export DISPLAY=:1
$ env EGL_PLATFORM=x11  anbox session-manager
Client is not authorized to connect to Server[ 2022-02-11 06:51:25] [daemon.cpp:61@Run] Failed to initialize SDL: No available video device
```

**What were you expecting?:**
To see if it is possible to solve the problem in current environment

**Additional info:**
Note that I am doing this on a virtura machine (something like EC2 in AWS) on the cloud, and managed to start the X server in vncserver. (I tried to start X server in console but it told me failed to find device, does that mean I nead an graphics card?)
Here is my vncservice config: 
```
cat ~/.vnc/xstartup
#!/bin/sh
xrdb $HOME/.Xresources
xsetroot -solid grey
xfce4-session &
#Xorg :1 &
```

Besides,  although it shows `binder: false` above, I did compile the module, `grep binder /proc/filesystems` returns `nodev	binder`, is that enough?

Thank you~  I have been stuck on this for more than a whole day

**Comment by Fuseteam** [comment]
Yes that should be enough for binder. The video device thing seems similar to what we get when launching session-manager too early in http://github.com/fuseteam/aind
